backend/src/routes/api/nb-events/index.ts

```diff
@@ -0,0 +1,21 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { getNotebookEvents } from './eventUtils';
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  fastify.get(
+    '/:notebookName',
+    async (
+      request: FastifyRequest<{
+        Params: {
+          notebookName: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      const params = request.params;
+      return getNotebookEvents(fastify, params.notebookName).catch((res) => {
+        reply.send(res);
+      });
```

Comment:
Please read https://www.fastify.io/docs/latest/Reference/Reply/#async-await-and-promises -- we do not need this `.catch`.

Reply:
Oh I thought that was the catch referenced by Juntao: 
> We don't need to catch the error here IMO. The error will be caught by the catch here https://github.com/opendatahub-io/odh-dashboard/pull/330/files#diff-97d842340dde487037d9347769f0bab0d4bf247ca2007270aa8a1c609c38573eR13-R15